.github/workflows/tests.yml

```diff
@@ -280,3 +280,54 @@ jobs:
           rm -rf qiskit_aer
           stestr run --slowest
         shell: bash
+  unit-tests-latest-qiskit-terra:
+    runs-on: ${{ matrix.os }}
+    needs: [sdist, lint]
+    timeout-minutes: 60
+    strategy:
+      matrix:
+        python-version: ["3.11"]
+        os: ["ubuntu-latest"]
```

Comment:
Are you thinking we should run this on every commit? Or was the eventual plan to run this job periodically when it's no longer a `WIP`? We can set this up in a new workflow file and create a nightly cron job to run this test if we want to do it periodically, see: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule

Reply:
I want to use this test case for testing deprecations and changes of Qiskit. I will remove this test case and only merge fixes in this PR.